src/dctkit/mesh/circumcenter.py

```diff
@@ -3,47 +3,49 @@
 import numpy.typing as npt
 from typing import Tuple
 
-# FIXME: pass only the coordinates of the nodes of the simplex or consider making this a
-# method of the SimplicialComplex class.
 
-
-def circumcenter(s: npt.NDArray,
+def circumcenter(S: npt.NDArray,
                  node_coords: npt.NDArray) -> Tuple[npt.NDArray, npt.NDArray]:
     """Compute the circumcenter of a given simplex. (Reference: Bell, Hirani,
```

Comment:
Change the description of the function: now we are not passing a given simplex, but a set of simplices.